discord_bot/message_sync.py

```
@@ -46,9 +46,19 @@ async def sync_recent_messages(channel, sync_limit: int = 200):
         
         if deleted_ids:
             logger.info(f"[Sync] Found {len(deleted_ids)} deleted messages in {channel_name}")
+            delete_success = 0
+            delete_failed = 0
             for msg_id in deleted_ids:
-                await delete_message(msg_id)
-                logger.debug(f"[Sync] Deleted message {msg_id} from DB")
+                try:
+                    await delete_message(msg_id)
+                    logger.debug(f"[Sync] Deleted message {msg_id} from DB")
```

Ruff (0.15.2)

[warning] Logging statement uses f-string

(G004)